src/server/handlers.rs

```diff
@@ -0,0 +1,240 @@
+use axum::Json;
```

Comment:
How would you see placing the `server` outside of the `src`? The reasoning would be to aim for not having any http server related dependencies in the pod2 library.

Reply:
In case we go with the http server approach, I think it makes sense to have it in a separate crate, so that when an external project imports the pod2 library does not import the server dependencies.

On the wasm vs httpserver approach, I'm not sure yet; I'm slightly inclined in favor of the wasm approach so that the webapp is a standalone app without requiring a remote server, but open to other reasonings.

Reply:
*separate crate, still in the same repo